Logging/.CSProject/anvil-csharp-logging.sln

```diff
@@ -1,12 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Anvil.CSharp.Logging", "Anvil.CSharp.Logging.csproj", "{C334863D-1273-44DA-AE08-E006B848D84C}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "anvil-csharp-logging", "anvil-csharp-logging.csproj", "{C334863D-1273-44DA-AE08-E006B848D84C}"
```

Comment:
Should the Anvil.Unity.Logging project be renamed to anvil-unity-logging to match? I can do that if it's not done yet

Reply:
Yep! https://github.com/decline-cookies/anvil-unity-core/pull/70